lib/ts/recipeImplementation.ts

```diff
@@ -44,6 +44,22 @@ export default function RecipeImplementation(recipeImplInput: {
         },
         addAxiosInterceptors: function(input: { axiosInstance: any; userContext: any }): void {
             logDebugMessage("addAxiosInterceptors: called");
+
+            if ((XMLHttpRequest as any).__interceptedBySuperTokens) {
+                console.warn(
+                    "Not adding axios interceptor since XMLHttpRequest is already added. This is just a warning."
+                );
+                console.warn("Our axios and XMLHttpRequest interceptors cannot be used at the same time.");
+                console.warn(
+                    "Since XMLHttpRequest is added automatically and supports axios by default, you can just remove addAxiosInterceptors from your code."
+                );
+                console.warn(
+                    "If you want to continue using our axios interceptor, you can override addXMLHttpRequestInterceptor with an empty function."
+                );
+
+                logDebugMessage("addAxiosInterceptors: not adding, because XHR interceptors are already in place");
+                return;
+            }
```

Comment:
I mean this is just a warning, and the Axios interceptor is already circumvented anyway if the XHR interceptor was added.
Still, I think this will have to be a breaking change in the website repo anyway (and BE is breaking for sure) but I'll check (and add an explicit FE test about continuing an old session)